components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2189,9 +2189,13 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 throw new Error(`No Stripe customer profile for '${attributionId}'`);
             }
 
-            await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
-            await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+            //TODO: feature flag with below
+            await this.billingService.createStripeSubscription({ attributionId, setupIntentId, usageLimit });
 
+            // await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
+            // await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+
+            // TODO maybe: Also move this call to setCostCenter to billingService.createStripeSubscription
```

Comment:
Please add the feature flag, or remove this change entirely.

Reply:
Yes - this is still in there only for testing, hence the [hold](https://github.com/gitpod-io/gitpod/pull/14044#issuecomment-1294917049).

Reply:
Feature flag will be in a follow-up PR.